ui/userguide/UserGuideFooter.jsx

```diff
@@ -31,7 +31,7 @@ const UserGuideFooter = function UserGuideFooter() {
           className="midgray"
           href="https://whatsdeployed.io/s/BIY/Mozilla/Treeherder"
         >
-          Whats Deployed?
+          What's Deployed?
```

Comment:
the problem here is we have a lint error:
```
error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;` (react/no-unescaped-entities) at ui/userguide/UserGuideFooter.jsx:34:15:
  32 |           href="https://whatsdeployed.io/s/BIY/Mozilla/Treeherder"
  33 |         >
> 34 |           What's Deployed?
     |               ^
  35 |         </a>
  36 |       </div>
  37 |     </div>
```

I am not sure of the solution, do we need escape characters and will that show up properly?

Reply:
We might be able to use `&apos;`? I'll try it.